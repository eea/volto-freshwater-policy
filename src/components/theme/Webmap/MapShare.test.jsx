import MapShare from './MapShare';
import React from 'react';
import { Provider } from 'react-intl-redux';
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';

const mockStore = configureStore();

test('renders map component', () => {
  const store = mockStore({
    userSession: { token: '1234' },
    intl: {
      locale: 'en',
      messages: {},
    },
  });

  const component = renderer.create(
    <Provider store={store}>
      <MapShare data={{ url: 'https://mapurl.com' }} />
    </Provider>,
  );
  const json = component.toJSON();
  expect(json).toMatchSnapshot();
});
