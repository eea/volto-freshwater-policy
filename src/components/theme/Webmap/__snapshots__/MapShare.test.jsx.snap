// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders map component 1`] = `
<button
  className="ui button toolbar-button"
  onBlur={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  title="Share"
>
  <svg
    className="icon"
    dangerouslySetInnerHTML={
      Object {
        "__html": undefined,
      }
    }
    onClick={null}
    style={
      Object {
        "fill": "currentColor",
        "height": "26px",
        "width": "auto",
      }
    }
    viewBox=""
    xmlns=""
  />
</button>
`;
